book/creating-a-basic-website/adding-a-theme.rst

```diff
@@ -95,5 +100,18 @@ The next table shows the standard commands available with its parameters.
 |         | h: the height of the new image    |
 +---------+-----------------------------------+
 
+Global Image Compression
+-----------------------
+Images will not get compressed by default, if you upload them. You can set the compression for images globally in the sulu.yml or seperat for each image format like in the example above.
```

Comment:
We've got a maximum line length of 80 characters. And the configuration should be in the `config.yml`, the `sulu.yml` should not be edited due to avoidance of merge conflicts.